bootstrap/kubeadm/main.go

```diff
@@ -166,6 +179,8 @@ func main() {
 		}()
 	}
 
+	tp := traceutil.NewProvider(tracingEndpoint, tracingSamplingRatePerMillion, "capi-kubeadm-bootstrap-controller-manager")
```

Comment:
Would it make sense to align the name with the user agent 2 lines below? (Question also applies to the other 3 controllers)

Reply:
Yes that would make sense.

Just fyi, this PR is currently only an experiment. I don't know when/if we will add tracing and it's entirely possible that we will add this feature to controller-runtime so we only have to enable it in CAPI